aspnetcore/signalr/messagepackhubprotocol.md

```diff
@@ -21,21 +21,21 @@ This article assumes the reader is familiar with the topics covered in [Get Star
 
 ## What is MessagePack?
 
-[MessagePack](https://msgpack.org/index.html) is a binary serialization format that is fast and compact. It's useful when performance and bandwidth are a concern because it creates smaller packets compared to [JSON](https://www.json.org/). Because it's a binary format, messages are unreadable when looking at network traces and logs unless the bytes are passed through a MessagePack parser. SignalR has added built-in support for the MessagePack format, and provides APIs for the client and server to use.
+[MessagePack](https://msgpack.org/index.html) is a binary serialization format that is fast and compact. It's useful when performance and bandwidth are a concern because it creates smaller messages compared to [JSON](https://www.json.org/). Because it's a binary format, messages are unreadable when looking at network traces and logs unless the bytes are passed through a MessagePack parser. SignalR has built-in support for the MessagePack format, and provides APIs for the client and server to use.
 
 ## Configure MessagePack on the server
 
 To enable the MessagePack Hub Protocol on the server, install the `Microsoft.AspNetCore.SignalR.Protocols.MessagePack` package in your app. Add `AddMessagePackProtocol` to the `AddSignalR` call to enable MessagePack support on the server.
 
```

Comment:
It would help beginners to specify that this code is in `Startup.cs`.

Maybe the last sentence could be something like "Within your `Startup.cs` file, add the messagepack protocol to SignalR:" and then have the code sample.

Reply:
Yeah, I always try to refer to the file because we don't know how experienced the reader is with ASP.NET Core :). Good note